tests/src/test/java/alluxio/shell/command/ComplexCommandTest.java

```diff
@@ -57,7 +56,6 @@ public void lsThenloadMetadataTest() throws IOException, AlluxioException {
     Assert.assertTrue(mFileSystem.getStatus(new AlluxioURI("/dir1")).isPersisted());
     // Load metadata
     mFsShell.run("loadMetadata", "/dir1");
-    Assert.assertEquals(ExceptionMessage.FILE_ALREADY_EXISTS.getMessage("/dir1") + "\n",
-        mOutput.toString());
+    Assert.assertEquals("", mOutput.toString());
```

Comment:
so we are changing the semantics of loadMetadata not to fail if the file already exists?